app/controllers/concerns/ip.rb

```diff
@@ -0,0 +1,34 @@
+module Ip
+  require "ipaddr"
+
+  extend ActiveSupport::Concern
+
+  def request_ip
+    anonymize_ip(request.ip)
+  end
+
```

Comment:
Should all this be private?

Reply:
`request_ip` can't be but everything else can be 👍